client/src/components/SectionTable/SectionTable.js

```diff
@@ -79,34 +97,73 @@ class SectionTable extends PureComponent {
                         <Fragment />
                     )}
                 </div>
-                <table className={classes.table}>
-                    <thead>
-                        <tr>
-                            {!this.props.colorAndDelete ? <th>Add</th> : <th style={{ border: 'none' }} />}
-                            <th>Code</th>
-                            <th>Type</th>
-                            <th>Instructors</th>
-                            <th>Times</th>
-                            <th>Places</th>
-                            <th>Enrollment</th>
-                            <th>Rstr</th>
-                            <th>Status</th>
-                        </tr>
-                    </thead>
-                    <tbody>
-                        {this.props.courseDetails.sections.map((section) => {
-                            return (
-                                <SectionTableBody
-                                    key={section.sectionCode}
-                                    section={section}
-                                    courseDetails={this.props.courseDetails}
-                                    term={this.props.term}
-                                    colorAndDelete={this.props.colorAndDelete}
-                                />
-                            );
-                        })}
-                    </tbody>
-                </table>
+
+                <TableContainer
+                    component={Paper}
+                    style={{ margin: '8px 0px 8px 0px' }}
+                    elevation={0}
+                    variant="outlined"
+                >
+                    <Table className={classes.table} size="small">
+                        <TableHead>
+                            <TableRow>
+                                <TableCell classes={{ sizeSmall: classes.cellPadding }} className={classes.row} />
```

Comment:
It seems like every TableCell has `classes={{ sizeSmall: classes.cellPadding }}` added to it.
Would it make sense to add `sizeSmall: classes.cellPadding` to `classes.row` and `classes.cell`, so we don't have to add the other classes statement everywhere?